src/GLideNUI/ConfigDialog.cpp

```diff
@@ -167,9 +167,21 @@ void ConfigDialog::_init(bool reInit, bool blockCustomSettings)
 	ui->fullScreenResolutionComboBox->setCurrentIndex(fullscreenMode);
 	ui->fullScreenRefreshRateComboBox->setCurrentIndex(fullscreenRate);
 
+	if (m_maxMSAA == 0 && config.video.maxMultiSampling == 0) {
```

Comment:
As I understand, we can't get valid config.video.maxMultiSampling until the first run of a game. It's not nice.

Reply:
That's correct, sadly that's a limitation I can't easily fix (to get it working on i.e mupen64plus), so yes, to get the correct max level of msaa, you need to run a game at least once and save at least once to get it 'cached' into the config file, how would ypu propose I fix that limitation then?

Reply:
I don't see anything better than my approach from WindowsWGL::maxMSAALevel()
that is create GL context, probe max msaa level, close the context.

Reply:
That's also not ideal because it causes a delay and that doesn't work on mupen64plus (nor linux)

Reply:
I agree, it is not very nice too, but small delay is better than incorrect max msaa level.
Ok, let it be as this. I'll probably modify it for Windows GUI.